jackal_navigation/launch/nav2.launch.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Clearpath Robotics, Inc.
+# Copyright (c) 2018 Intel Corporation
```

Comment:
Why  the change here?

Reply:
Here, I changed the nav2.launch.py file to the original `navigation.launch.py` file in the `nav2_bringup` package (found [here](https://github.com/ros-planning/navigation2/blob/foxy-devel/nav2_bringup/bringup/launch/navigation_launch.py) because the default `nav2.launch.py` didn't seem to be working.